wandb/sdk_py27/wandb_settings.py

```diff
@@ -1039,6 +1042,14 @@ def _apply_init(self, args):
                     wandb.termwarn(
                         "Ignored wandb.init() arg %s when running a sweep" % key
                     )
+        if self.launch:
+            for key in ("project", "entity", "id"):
+                val = args.pop(key, None)
+                if val:
+                    wandb.termwarn(
+                        "Ignored wandb.init() arg %s when running running from launch"
```

Comment:
clearer error message that makes it clear we're only overriding project/entity fields would be good here